server/svix-server/tests/e2e_endpoint.rs

```diff
@@ -669,16 +669,15 @@ async fn test_custom_endpoint_secret() {
 }
 
 #[tokio::test]
-#[ignore]
 async fn test_invalid_endpoint_secret() {
     let (client, _jh) = start_svix_server();
 
     let app_id = create_test_app(&client, "app1").await.unwrap().id;
 
-    let secret_invalid_prefix = "hwsec_C2FVsBQIhrscChlQIMV+b5sSYspob7oD".to_owned();
```

Comment:
Why remove the invalid prefix test?

Reply:
It was removed because of the way deserialization currently works. It doesn't validate the prefix but rather just strips off the length of the prefix from the incoming secret and decodes the rest of it. We could potentially add an additional equality check of the prefix itself during deserialization.